data/crac/crac-loopflow-extension/src/main/java/com/farao_community/farao/data/crac_loopflow_extension/JsonCnecLoopFlowExtensionSerializer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+package com.farao_community.farao.data.crac_loopflow_extension;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Cnec;
+import com.farao_community.farao.data.crac_api.ExtensionsHandler;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.google.auto.service.AutoService;
+
+import java.io.IOException;
+
+/**
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
+ */
+@AutoService(ExtensionsHandler.ExtensionSerializer.class)
+public class JsonCnecLoopFlowExtensionSerializer implements ExtensionsHandler.ExtensionSerializer<Cnec, CnecLoopFlowExtension> {
+
+    @Override
+    public void serialize(CnecLoopFlowExtension cnecLoopFlowExtension, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+        jsonGenerator.writeStartObject();
+        jsonGenerator.writeNumberField("inputThreshold", cnecLoopFlowExtension.getInputThreshold());
+        jsonGenerator.writeObjectField("inputThresholdUnit", cnecLoopFlowExtension.getInputThresholdUnit().name());
+        jsonGenerator.writeEndObject();
+    }
+
+    @Override
+    public CnecLoopFlowExtension deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException {
+        double inputThreshold = 100.0;
+        Unit unit = Unit.PERCENT_IMAX;
```

Comment:
I think it's better to put these default values in the constructor of the extension instead

Reply:
I don't really like the idea of a default constructor because it would require to define method `setInputThreshold()` and `setInputThresholdUnit()`, while those values should be final as we do not want to change them during the RAO.

Actually i could be more demanding here and impose that the two values are given in the JSON file (and so remove the default value). Anyway if the JSON crac has been provided by the JsonCracExporter, it will always contains the two values.

Reply:
Ok then just define final static variables for these values ! ;)